src/actions/actions.rs

```diff
@@ -2,8 +2,10 @@
 #[derive(Debug, PartialEq, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub enum ActionValue {
-    ReviewRequested,
     Synchronize,
+    Opened,
+    Reopened,
+    Closed,
```

Comment:
One question: in case somebody defines non-default configuration options in his `LabHub.toml`, would this still work or do all possible options need to be mentioned here?

Reply:
Yes, it will still work. The TOML file may contain lots of entries, but only those which are actual actions will be taken into account.

Reply:
Ok, I see what you mean. It might be reasonable to add the complete list since this is no config file.